deploy/dashboards/grafana-dashboard-provisioning.configmap.yaml

```
@@ -599,7 +599,7 @@ data:
                 "type": "prometheus",
                 "uid": "PDD8BE47D10408F45"
               },
-              "expr": "sum(rate(provisioning_source_availability_check_request_total{error=\"false\"}[$__range]))/sum(rate(provisioning_source_availability_check_request_total[$__range]))",
+              "expr": "sum(rate(provisioning_source_availability_check_request_total[$__range]))/sum(rate(provisioning_http_request_total{path="/api/provisioning/v1/availability_status/sources/"}[$__range]))",
```

I guess we decided not to use the request count as the total, right?
```suggestion
              "expr": "sum(rate(provisioning_source_availability_check_request_total[$__range]))/sum(rate(provisioning_source_availability_check_request_enqueued_total[$__range]))",
```